sys/hoon.hoon

```diff
@@ -3418,6 +3418,7 @@
   |_  a/@
   ++  rad                                               ::  random in range
     |=  b/@  ^-  @
+    ~_  leaf+"rad-zero"
```

Comment:
This looks to be missing the assertion that was present in the earlier PR.

(BTW, you can force-push to a branch and arbitrarily change a pull-request. You can even switch to a different target branch. You shouldn't need to open a new PR to replace the contents.)

Reply:
Yikes, that was an unfortunate omission.  Fixed now.